src/tools/compiletest/src/runtest.rs

```diff
@@ -2987,8 +2987,7 @@ impl<'test> TestCx<'test> {
             self.safe_revision().map_or("nightly".into(), |path| path.to_owned() + "-nightly");
         let compare_dir = output_base_dir(self.config, self.testpaths, Some(&suffix));
         // Don't give an error if the directory didn't already exist
```

Comment:
This comment can probably be removed/moved to the function

Reply:
Good catch. Let's remove it since it doesn't add much.